client/src/style/scss/ui.scss

```diff
@@ -473,6 +473,39 @@ $ui-margin-horizontal-large: $margin-v * 2;
     }
 }
 
+/* Heading Sizes */
+.h-xl {
+    font-size: $h1-font-size;
+}
+
+.h-lg {
+    font-size: $h2-font-size;
+}
+
+.h-md {
+    font-size: $h3-font-size;
+}
+
+.h-sm {
+    font-size: $h4-font-size;
+}
+
+.h-text {
+    font-size: $font-size-base;
+}
+
```

Comment:
This decoupling of header font-sizes from header-tags which in are then used semantically is awesome. Thank you also for the documentation.